packages/react/src/molecules/pickers/radio/simple.tsx

```diff
@@ -60,22 +59,32 @@ const OptionsRenderer = ({
     <div className="flex items-center">
       <TextInputBase
         labelBaseProps={{
-          label:"",
+          label: "",
         }}
         htmlFor=""
         id={value.value}
         name={name}
         type="radio"
         checked={found === true}
         onChange={() => {}}
-        textInputBaseClasses={radioBoxStyleClasses ? { ...theme.radioBoxStyleClasses, ...radioBoxStyleClasses} : theme.radioBoxStyleClasses}
+        textInputBaseClasses={
+          radioBoxStyleClasses
+            ? { ...theme.radioBoxStyleClasses, ...radioBoxStyleClasses }
+            : theme.radioBoxStyleClasses
+        }
       />
 
       <LabelBase
         htmlFor={value.value}
         label={value.label}
         labelBaseClasses={
-          theme.pickerRadioSimpleClasses.optionClasses.labelBaseClasses
+          radioBoxStyleClasses?.labelBaseClasses
```

Comment:
@ShreyDhyani @Subbu-betalectic I think we can improve this by actually passing labelBaseProps rather than these styles I guess right?

The code here is getting complex, please suggest.